config/jobs/kubernetes/test-infra/test-infra-presubmits.yaml

```diff
@@ -205,7 +205,6 @@ presubmits:
     trigger: "/test pull-test-infra-verify-codegen"
     run_if_changed: '(prow/(apis|client)|vendor)/.*$'
```

Comment:
Also if we believe we have it working now let's just run it on every PR.

I shouldn't have to modify this regex every time I add a new controller somewhere in the codebase.